src/web/client/src/components/Login/Login.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+import Login from "./Login";
+
+describe("Login component", () => {
+  let loginWrapper: any;
+  let onAuthChangeMock: jest.Mock;
+  let historyMock: object;
+  let eventMock: any;
```

Comment:
just something to highlight about this `any`

Reply:
It's supposed to have a FormEvent type but it requires all the fields of the FormEvent type, and there is a lot of them.